sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -48,6 +54,12 @@ impl InternalError {
 
 fn run(operation: PluginOp) -> Result<std::process::ExitStatus, InternalError> {
     let status = match operation {
+        PluginOp::Type {} => {
+            println!("debian");
```

Comment:
Why is `apt` plugin reporting it's type as `debian`?

Reply:
Isn't that the module type supported by this plugin? Or was it supposed to be `apt`? All our existing specs had this `debian` type mentioned in the URL scheme and all. So I thought that's what we wanted to use as well.

Reply:
After further discussions, it has been decided that the plugin must report the package manager type (`apt` in this case) instead of the package type itself here, as a single package type could have multiple package managers that can handle that type. For example, a debian package can be handled by both `apt` and `dpkg` package managers.